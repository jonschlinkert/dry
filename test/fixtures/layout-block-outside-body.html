{% layout "layout-default.html" %}

This is root-text-node content from "layout-block-outside-body.html".

{% block "head" %}
head: "layout-block-outside-body.html"
{% endblock %}
