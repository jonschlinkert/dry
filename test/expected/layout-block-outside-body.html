<!DOCTYPE html>
<html lang="en">
  
head: "layout-block-outside-body.html"
  <body>
    
This is root-text-node content from "layout-block-outside-body.html".

    
  </body>
</html>